drivers/scheduler/k8s/specs/elasticsearch-crd-webhook/cr-elasticsearch.yaml

```diff
@@ -0,0 +1,23 @@
+# Source: https://portworx.atlassian.net/wiki/spaces/PE/pages/2548171181/App+installation+via+Operator#Elasticsearch
+# Source: https://portworx.atlassian.net/wiki/spaces/PE/pages/2548171181/App+installation+via+Operator#Elasticsearch
```

Comment:
Can we not add internal links in public repo?

Reply:
✅

Reply:
I'm not sure why you're seeing it, but you must have clicked the link to the file (`..../cr-elasticsearch.yaml`) at the top of this conversation, which _will_ take you the part of the file where you commented, **but** it takes you to the commit on which you commented (which is the older commit). Just view the diffs and files from the latest commit; you'll see the change ✅